src/components/features/home/contact/contact-form/contact-form.scss

```diff
@@ -0,0 +1,89 @@
+
+input,
+.input__message {
+  border: 1px solid #000;
+  width: 90%;
+  height: 2.4em;
+  box-sizing: border-box;
+  padding-left: 1em;
+}
+
+::placeholder {
+  font-family: 'Raleway';
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.form__container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.input__message {
+  color: #000;
+  font-family: 'Raleway';
+  height: 14em;
+  resize: none;
+}
+.input__checkbox {
+  width: 1em;
+  height: 1em;
+}
+.checkbox__label {
+  font-size: 0.7em;
+}
+.checkbox__row {
+  display: flex;
+  flex-direction: row;
+  width: 90%;
+}
+.form__button {
+  background-color: #46b8a7;
+  color: #fff;
+  width: 7em;
+  border-radius: 3.5em;
+  -webkit-border-radius: 3.5em;
+  -moz-border-radius: 3.5em;
+  -ms-border-radius: 3.5em;
+  -o-border-radius: 3.5em;
+  margin-bottom: 2em;
+}
+
+.form__error {
+  color: #cc0000;
```

Comment:
Uwzględniłabym ten kolor w zmiennych w pliku `styles/_variables.scss` i użyła go tutaj, będę bardzo wdzięczna jeśli będziesz o tym pamiętać, bo to bardzo się przyda żeby spójność kolorów była na całej stronce - jakbym robiła formularze to chciałabym wykorzystać ten sam kolor jak tutaj.